src/main/java/com/aspire/student/controller/StudentController.java

```diff
@@ -148,7 +148,8 @@ public void saveStudent(ActionRequest actionRequest, ActionResponse actionRespon
     try {
       student = studentService.saveStudent(student);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Introduce a new variable instead of reusing the parameter "student". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1226)